pages/agent/v3/help/_start.md

```diff
@@ -89,8 +89,16 @@ The agent will run any jobs within a PTY (pseudo terminal) if available.
+<tr id="signal-grace-period-seconds"><th><code>--signal-grace-period-seconds value</code> <a class="Docs__attribute__link" href="#signal-grace-period-seconds">#</a></th><td><p>The number of seconds given to a subprocess to handle being sent `cancel-signal`. After this period has elaspsed, SIGKILL will be sent. (default: 0)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_SIGNAL_GRACE_PERIOD_SECONDS</code></p></td></tr>
 <tr id="tracing-backend"><th><code>--tracing-backend value</code> <a class="Docs__attribute__link" href="#tracing-backend">#</a></th><td><p>Enable tracing for build jobs by specifying a backend, "datadog" or "opentelemetry"<br /><strong>Environment variable</strong>: <code>$BUILDKITE_TRACING_BACKEND</code></p></td></tr>
 <tr id="tracing-service-name"><th><code>--tracing-service-name value</code> <a class="Docs__attribute__link" href="#tracing-service-name">#</a></th><td><p>Service name to use when reporting traces. (default: "buildkite-agent")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_TRACING_SERVICE_NAME</code></p></td></tr>
+<tr id="job-verification-key-path"><th><code>--job-verification-key-path value</code> <a class="Docs__attribute__link" href="#job-verification-key-path">#</a></th><td><p>Path to a file containing a verification key. Passing this flag enables job verification. For hmac-sha256, the raw file content is used as the shared key<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_JOB_VERIFICATION_KEY_PATH</code></p></td></tr>
+<tr id="job-verification-jwks-path"><th><code>--job-verification-jwks-path value</code> <a class="Docs__attribute__link" href="#job-verification-jwks-path">#</a></th><td><p>Path to a file containing a JSON Web Key Set (JWKS), used to verify job signatures.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_JWKS_FILE_PATH</code></p></td></tr>
+<tr id="job-signing-jwks-path"><th><code>--job-signing-jwks-path value</code> <a class="Docs__attribute__link" href="#job-signing-jwks-path">#</a></th><td><p>Path to a file containing a signing key. Passing this flag enables pipeline signing for all pipelines uploaded by this agent. For hmac-sha256, the raw file content is used as the shared key<br /><strong>Environment variable</strong>: <code>$BUILDKITE_PIPELINE_UPLOAD_JWKS_FILE_PATH</code></p></td></tr>
+<tr id="job-signing-algorithm"><th><code>--job-signing-algorithm value</code> <a class="Docs__attribute__link" href="#job-signing-algorithm">#</a></th><td><p>The algorithm to use when signing pipelines. Must be a valid RF7518 JWA algorithm. Required when using a JWKS, and the given key doesn't have an alg parameter<br /><strong>Environment variable</strong>: <code>$BUILDKITE_PIPELINE_UPLOAD_SIGNING_ALGORITHM</code></p></td></tr>
```

Comment:
Does this say, "JSON Web Algorithm algorithm?"

Reply:
Even worse, it's "JSON Web Algorithms algorithm".